.claude/skills/productize-chart-env/scripts/gen-schema.py

```
@@ -161,6 +177,17 @@ def parse_key_list(path):
     return keys
 
 
+def _allow(keys):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_allow`

(ANN202)